### PR TITLE
add missing space settings title

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1584

* Add missing title + fixed styles in corresponding places its used 
image
* Fixed no-events message display
Before:
image
After:
image

After:

### CODE REVIEW

File: src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import "scss/constants.scss";
 
+$no-events-margin-top: 100px;
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss:3]: Odd number to have, more so that it's combined with auto and 0

Comment (goran-peoski-work) [reply src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss:3]: To center an element, what usually works is to give the parent:
```css
  .parent {
    display: grid;
    place-content: center;  /* one of those place- props */
  }
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss (hunk 1 above)
Comment (alexey-petrov) [reply src/components/organisms/AdminVenueView/components/EventsView/EventsView.scss]: got rid of this margin in favor of `flex`
grid didn't really work for me here 3c4ede1

**Pull request merged**